client/pages/receptionistDashboard.jsx

```diff
@@ -26,6 +26,8 @@ const ReceptionistDashboard = () => {
     const [showInfoAlert, setShowInfoAlert] = useState(false);
     const [errorMessage, setErrorMessage] = useState("");
     const [showScanner, setShowScanner] = useState(false);
+    const [visitModalData, setVisitModalData] = useState("");
```

Comment:
string won't cut it either, I am not sure what to make it either, I have a similar problem on the bulk-sign in branch. I will change it once I know what to do. Please remind me